charts/app-config/values-production.yaml

```diff
@@ -2469,6 +2469,49 @@ govukApplications:
               name: publishing-api-bigquery
               key: client-secret
 
+  - name: publishing-api-read-replica
+    repoName: publishing-api
+    helmValues:
+      arch: arm64
+      appResources:
+        limits:
+          cpu: 2
+          memory: 1.5Gi
+        requests:
+          cpu: 20m
+          memory: 768Mi
+      nginxClientMaxBodySize: 2M
+      dbMigrationEnabled: true
```

Comment:
Does this need to be disabled since the migrations will run in the main app deployment?